src/components/GridCard.js

```diff
@@ -34,7 +34,12 @@ const GridCard = (props) => (
       alignSelf="start"
       width="auto"
     >
-      <Anchor color="black" margin="small" color="rgb(0,0,0)">
+      <Anchor
+        color="black"
+        margin="small"
+        color="rgb(0,0,0)"
+        href={props.openLink}
+      >
```

Comment:
Why does `color` appear twice?
I think Anchor shouldn't have a color at all, @jcfilben did you check it with Marisa?

Reply:
Good catch, we already checked with Marisa and yeah the anchor shouldn't have color at all. It is fixed in the most recent push

Reply:
Yes I asked Marisa and she talked with Greg this morning. She said she is fine with the anchor links being the grommet theme colors, I am working on a PR to change the anchors so the color isn't being set to black